File: app_java/src/main/java/vlab/android/architecture/util/TextValidator.java
package vlab.android.architecture.util;

/**
 * Created by Vinh Tran on 10/24/18.
 * just demo
 */
public class TextValidator {
    public static boolean isUserNameValid(String userName){
        return userName != null && !"".equals(userName);
    }

    public static boolean isPwdValid(String pwd) {
        return pwd != null && !"".equals(pwd);
    }
}

File: app_kotlin/src/main/java/vlab/android/architecture/util/TextValidator.kt
package vlab.android.architecture.util

/**
 * Created by Vinh Tran on 10/24/18.
 * just demo
 */
object TextValidator {
    fun isUserNameValid(userName: String?): Boolean {
        return userName != null && "" != userName
    }

    fun isPwdValid(pwd: String?): Boolean {
        return pwd != null && "" != pwd
    }
}
